app/services/renderer_methods.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# macros to fetch information about invoked jobs
+module RendererMethods
+  extend ActiveSupport::Concern
+
+  def find_job_invocation_by_id(job_id, preload: nil)
```

Comment:
Actualy why do we use this, instead of using `JobInvocation.find(id)` in the template?

Reply:
This method has similar reason like the upper one - I like to avoid user to use methods from AR.

Reply:
This method is no longer used IIRC